website/README.md

```diff
@@ -0,0 +1,30 @@
+# React + TypeScript + Vite
```

Comment:
Please update the readme to actually reflect the project with a short installation guide